src/postprocess/band_structure.jl

```diff
@@ -3,9 +3,9 @@ include("../external/pymatgen.jl")
 
 # Functionality for computing band structures, mostly using pymatgen
 
-function high_symmetry_kpath(basis, kline_density, composition...)
+function high_symmetry_kpath(basis, kline_density, atoms)
```

Comment:
I guess you could add a version without the `atoms` (getting them from the `basis`) for convenience.